scripts/postprocess.py

```diff
@@ -353,13 +353,23 @@ def parse_jobs(data_dirs, distinguish_dir=False):
         data_dirs = [data_dirs]
     data = []
     for data_dir in data_dirs:
+        bench_name_postfix = os.path.basename(os.path.dirname(data_dir))
```

Comment:
Possible alternative: strip the common prefix from all `data_dirs`. If the basename happens to be the but some other parent component of the path is different, then you'd still be able to distinguish the paths. Worst case is you still end up with the full path in the name, but at least they'll be different. However, this is enough of a corner case that I don't care too much and would be happy with a simple solution (like now) here.

Reply:
possible alternative:
`--path=<path>[:<label>]`
In case label is not specified path is used as label.
```
for data_dir_label in data_dirs:
    dir_label = data_dir_label.split(":")
    data_dir = dir_label[0]
    if len(dir_label) == 1:
        label = data_dir
    else:
        label = dir_label[1]
```

Reply:
probably with @rasolca solution there is a bit more control without too much complication.

Reply:
@rasolca's option sounds pretty nice to me...